### PR TITLE
[zh] sync audit task page

### PR DESCRIPTION
**Pull request by tengqm**
Related: #26178

### CODE REVIEW

**Review by Aut0R3V — COMMENTED** [review]
approving for the English version

File: content/zh/docs/tasks/debug-application-cluster/audit.md [hunk 1]
```diff
@@ -390,239 +400,36 @@ and in the logs to monitor the state of the auditing subsystem.
...
-## Log Collector Examples
-
-### Use fluentd to collect and distribute audit events from log file
-
-[Fluentd](https://www.fluentd.org/) is an open source data collector for unified logging layer.
-In this example, we will use fluentd to split audit events by different namespaces.
--->
-## 日志收集器示例
-
-### 使用 fluentd 从日志文件中选择并且分发审计日志
```
Comment (Aut0R3V) [inline content/zh/docs/tasks/debug-application-cluster/audit.md:418]: why is all of this commented out?

Comment (tengqm) [reply content/zh/docs/tasks/debug-application-cluster/audit.md:418]: IT was removed from English version possibly because fluentd is 3rd-party content.

**Pull request merged**